usr/share/rear/lib/config-functions.sh

```diff
@@ -25,7 +25,8 @@ function SetOSVendorAndVersion () {
             grep -q -i 'debian' /etc/os-release && OS_VENDOR=Debian
             grep -q -i -E '(ubuntu|linuxmint)' /etc/os-release && OS_VENDOR=Ubuntu
             grep -q -i 'arch' /etc/os-release && OS_VENDOR=Arch
-            OS_VERSION=$(grep "^VERSION_ID=" /etc/os-release | cut -d\" -f2 ) # 7
+            local version_id=$(grep "^VERSION_ID=" /etc/os-release | cut -d\" -f2 ) # 7
```

Comment:
I think this is a better idiom to read os-release:
```sh
...=$(source /etc/os-release; echo ${VERSION_ID})
```
of course, you don't have to change it in this PR as the topic is unrelated.

Reply:
Agree, just that the other lines where also not doing that :shrug:

Reply:
There actually is a good reason for this awkward way of reading the file: hostile code in it won't affect ReaR. To follow a "defensive" programming style I'd prefer to leave it this way and not switch to `source` the file.